marklogic-data-hub-central/ui/src/pages/Load.tsx

```diff
@@ -38,8 +36,6 @@ const Load: React.FC = () => {
   };
 
   useEffect(() => {
-    setPage(location.state?.page);
```

Comment:
Pagination looks good but I see you undid some of my recent work that was needed for preserving pagination when cancelling 'Add to New Flow'. Was it a necessary change for this ticket?

https://drive.google.com/file/d/1VIKQ0OfuiTSqONtrZfP_Ra28Kpyxc3ZN/view

Reply:
@briantang Can you please review again.